DEVELOPING.md

```diff
@@ -0,0 +1,137 @@
+# Builder Services Development Environment
+
+## Overview
+
+This document captures the steps to start and run a Builder environment for development. The builder environment includes the builder services, as well as the depot web site.
+
+There are several ways of creating a Builder dev environment - but supporting all operating systems and environments has proven to be untenable. This document includes one officially supported way of creating a Builder dev environment, and links to unsupported ways of creating the dev environment that you may use at your own risk.
+
+## Pre-Requisites
+
+### Ports required
+1. 9636 - Intra-supervisor communication
+1. 80 - Web app
+1. 9631 - supervisor api port
+1. 5432 - posgres port
+
+### Checkout
+1. If you are developing on Linux
+1. 1. Ensure you have curl
+1. 1. `git clone https://github.com/habitat-sh/builder.git /src/builder`
```

Comment:
I'm guessing you want:
```
* If you are developing on Linux
  * Ensure you have curl
  * `git clone https://github.com/habitat-sh/builder.git /src/builder`
* If you are developing on a Mac, `git clone https://github.com/habitat-sh/builder.git`
```
which renders as:
* If you are developing on Linux
  * Ensure you have curl
  * `git clone https://github.com/habitat-sh/builder.git /src/builder`
* If you are developing on a Mac, `git clone https://github.com/habitat-sh/builder.git`

Since I don't seem to be able to get numbered sublists to work right